Gmplot-Track the Route/main.py

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
upload the CSV to be used in the script for testing

Reply:
I think my code has some bugs i will resolve them in a day or two and will notify you.

Reply:
Okay, make sure to update it by tomorrow

Reply:
Actually i m out of town these days i will be back in 3 days so it would be great sir if you wait , please

Reply:
Oh okay, I will put the hold label for now.